src/foam/nanos/crunch/ruler/ConfigureUCJExpiryOnGranted.js

```diff
@@ -79,7 +79,6 @@ foam.CLASS({
             logger.debug(this.getClass().getSimpleName(), "ucj.setExpiry()", ucj);
             if ( junctionExpiry != null && ( data instanceof RenewableData ) ) {
               ((RenewableData) data).setRenewable(false);
-              ((RenewableData) data).setReviewed(false);
```

Comment:
Just commenting that I agree with this. The configureUcjExpiryOnGranted - should not be specific to one capability.